lib/libutee/tee_api.c

```diff
@@ -10,10 +10,17 @@
 #include <types_ext.h>
 #include <user_ta_header.h>
 #include <utee_syscalls.h>
+#if defined(CFG_OCALL)
+#include "pta_ocall.h"
+#endif
 #include "tee_api_private.h"
 
 static const void *tee_api_instance_data;
 
+#if defined(CFG_OCALL)
+TEE_TASessionHandle tee_api_ocall_session;
```

Comment:
`static`?

Reply:
This is also linked against in `user_ta_entry.c` to tear down the session after the TA exits.

Reply:
Right.